### PR TITLE
Add additional events statements metrics

### PR DESCRIPTION
**Pull request by SuperQ**
* sort merge passes - lots of merge passes can indicate too small a
  sort_buffer_size
* sort rows - total number of rows sorted, useful for finding expensive
  statements
* no index used - statements that cause full table scans

### CODE REVIEW

File: mysqld_exporter.go [hunk 1]
```diff
@@ -194,6 +194,9 @@ const (
 		    SUM_ROWS_EXAMINED,
 		    SUM_CREATED_TMP_DISK_TABLES,
 		    SUM_CREATED_TMP_TABLES
+		    SUM_SORT_MERGE_PASSES
```
Comment (brian-brazil) [inline mysqld_exporter.go]: Are you missing commas here?

Comment (SuperQ) [inline mysqld_exporter.go]: I was :)

**Comment by SuperQ** [comment]
@brian-brazil Running a test of this, already found a bunch of schemas missing really basic indexes. :-)

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**